doc/nrf/device_guides/working_with_nrf/nrf70/developing/fw_patches_ext_flash.rst

```diff
@@ -0,0 +1,140 @@
+.. _ug_nrf70_developing_fw_patch_ext_flash:
+
+Firmware patches in the external memory
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide explains how you can use the nRF70 Series firmware patches in the external memory feature.
+
+Overview
+********
+
+By default the nRF70 Series firmware patches are built as part of the nRF Wi-Fi host driver code, residing in on-chip memory.
+The firmware patches include code that is executed on the nRF70 Series device.
+The size of the firmware patches may be considerably large, which limits the amount of on-chip code memory available for the user application.
+In order to increase the amount of on-chip memory available for user applications, the nRF70 Series Wi-Fi driver supports the option of using external memory, if that is available.
+
+Prerequisites
+=============
+
+Before using this feature, make sure that the following prerequisites are completed:
+
+ * The external memory must be available and configured in the devicetree.
+ * The external memory must be large enough to store the firmware patches, taking into consideration also the patch upgrade requirement, for example, Device Firmware Update (DFU).
+   The maximum size of all the firmware patches combined is 128 KB.
+
+.. note::
+    Currently, in Nordic Semiconductor SoCs, access to external memory and the Execute in Place (XIP) feature are available through the QSPI interface only.
+
+Supported platforms
+===================
+
+The following platforms are supported:
+
+* nRF5340 SoC or nRF5340 DK with nRF7002 EK as a shield
+* nRF52840 SoC or nRF52840 DK with nRF7002 EK as a shield
+
+.. note::
+    Due to limitations mentioned above, the nRF7002 DK is not supported as it needs access to external memory through the SPI interface.
+
+Available options
+*****************
+
+There are two ways to use the firmware patches in the external memory feature:
+
+* XIP supported external memory
```

Comment:
"XIP supported" sounds weird. I think it would be better to use something like "using XIP access" and "using QSPI transfers to RAM" or "There are two options [..] that can be used, depending on whether XIP access is available or not".